### ISSUE TITLE
redis-server crash at RedisAI queuePush while on AI.MODELRUN ...

### ISSUE THREAD

**Issue by filipecosta90** [RedisAI/RedisAI#205]
redis-server crash at RedisAI queuePush while on AI.MODELRUN ...

infrastructure/OS details:
OS Ubuntu 18.04
```
NAME="Ubuntu"
VERSION="18.04.3 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.3 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```
Kernel
```
4.15.0-1045-aws #47-Ubuntu SMP Fri Aug 2 13:50:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```
Infra
```
$lscpu
Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
CPU(s):              8
On-line CPU(s) list: 0-7
Thread(s) per core:  1
Core(s) per socket:  8
Socket(s):           1
NUMA node(s):        1
Vendor ID:           GenuineIntel
CPU family:          6
Model:               85
Model name:          Intel(R) Xeon(R) Platinum 8124M CPU @ 3.00GHz
Stepping:            4
CPU MHz:             3401.594
BogoMIPS:            5999.96
Hypervisor vendor:   KVM
Virtualization type: full
L1d cache:           32K
L1i cache:           32K
L2 cache:            1024K
L3 cache:            25344K
NUMA node0 CPU(s):   0-7
Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss ht syscall nx pdpe1gb rdtscp lm constant_tsc rep_good nopl xtopology nonstop_tsc cpuid aperfmperf tsc_known_freq pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single pti fsgsbase tsc_adjust bmi1 hle avx2 smep bmi2 erms invpcid rtm mpx avx512f avx512dq rdseed adx smap clflushopt clwb avx512cd avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves ida arat pku ospke
```

Crash file:

```
16136:C 05 Sep 2019 15:57:00.117 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
16136:C 05 Sep 2019 15:57:00.117 # Redis version=5.0.5, bits=64, commit=c696aebd, modified=0, pid=16136, just started
16136:C 05 Sep 2019 15:57:00.117 # Configuration loaded
16137:M 05 Sep 2019 15:57:00.118 # Server initialized
16137:signal-handler (1567699210) Received SIGTERM scheduling shutdown...
16137:signal-handler (1567699358) Received SIGTERM scheduling shutdown...
16137:signal-handler (1567699363) Received SIGTERM scheduling shutdown...
16849:C 05 Sep 2019 16:09:44.787 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
16849:C 05 Sep 2019 16:09:44.787 # Redis version=5.0.5, bits=64, commit=c696aebd, modified=0, pid=16849, just started
16849:C 05 Sep 2019 16:09:44.787 # Configuration loaded
16850:M 05 Sep 2019 16:09:44.788 # Could not create server TCP listening socket *:6379: bind: Address already in use
16855:C 05 Sep 2019 16:10:05.909 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
16855:C 05 Sep 2019 16:10:05.909 # Redis version=5.0.5, bits=64, commit=c696aebd, modified=0, pid=16855, just started
16855:C 05 Sep 2019 16:10:05.909 # Configuration loaded
16856:M 05 Sep 2019 16:10:05.910 # Server initialized


=== REDIS BUG REPORT START: Cut & paste starting from here ===
16856:M 05 Sep 2019 16:11:11.479 # Redis 5.0.5 crashed by signal: 11
16856:M 05 Sep 2019 16:11:11.479 # Crashed running the instruction at: 0x7f4a5f863432
16856:M 05 Sep 2019 16:11:11.479 # Accessing address: (nil)
16856:M 05 Sep 2019 16:11:11.479 # Failed assertion: <no assertion failed> (<no file>:0)

------ STACK TRACE ------
EIP:
/root/redisai_src/build/redisai.so(queuePush+0x7e)[0x7f4a5f863432]

Backtrace:
redis-server *:6379(logStackTrace+0x5a)[0x562abb93c37a]
redis-server *:6379(sigsegvHandler+0xb1)[0x562abb93cb31]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x12890)[0x7f4a620a2890]
/root/redisai_src/build/redisai.so(queuePush+0x7e)[0x7f4a5f863432]
/root/redisai_src/build/redisai.so(RedisAI_ModelRun_RedisCommand+0xb0f)[0x7f4a5f86735d]
redis-server *:6379(RedisModuleCommandDispatcher+0x68)[0x562abb96a128]
redis-server *:6379(call+0xb5)[0x562abb8f5dc5]
redis-server *:6379(processCommand+0x36f)[0x562abb8f64cf]
redis-server *:6379(processInputBuffer+0x171)[0x562abb906fb1]
redis-server *:6379(aeProcessEvents+0x150)[0x562abb8ef800]
redis-server *:6379(aeMain+0x2b)[0x562abb8efbeb]
redis-server *:6379(main+0x4f4)[0x562abb8ec794]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xe7)[0x7f4a61cc0b97]
redis-server *:6379(_start+0x2a)[0x562abb8ec9ea]

------ INFO OUTPUT ------
# Server
redis_version:5.0.5
redis_git_sha1:c696aebd
redis_git_dirty:0
redis_build_id:89f8911bbfe9347d
redis_mode:standalone
os:Linux 4.15.0-1045-aws x86_64
arch_bits:64
multiplexing_api:epoll
atomicvar_api:atomic-builtin
gcc_version:7.4.0
process_id:16856
run_id:327afa9a2df9bf7c22ffe55c23ab4e1f1899e0e2
tcp_port:6379
uptime_in_seconds:66
uptime_in_days:0
hz:10
configured_hz:10
lru_clock:7418783
executable:/root/redisai_src/redis-server
config_file:/root/redis_src/redis.conf

# Clients
connected_clients:16
client_recent_max_input_buffer:1013
client_recent_max_output_buffer:0
blocked_clients:15

# Memory
used_memory:747957728
used_memory_human:713.31M
used_memory_rss:835117056
used_memory_rss_human:796.43M
used_memory_peak:747957728
used_memory_peak_human:713.31M
used_memory_peak_perc:100.09%
used_memory_overhead:33982016
used_memory_startup:792464
used_memory_dataset:713975712
used_memory_dataset_perc:95.56%
allocator_allocated:747468264
allocator_active:747958272
allocator_resident:769794048
total_system_memory:43243544576
total_system_memory_human:40.27G
used_memory_lua:37888
used_memory_lua_human:37.00K
used_memory_scripts:0
used_memory_scripts_human:0B
number_of_cached_scripts:0
maxmemory:0
maxmemory_human:0B
maxmemory_policy:noeviction
allocator_frag_ratio:1.00
allocator_frag_bytes:490008
allocator_rss_ratio:1.03
allocator_rss_bytes:21835776
rss_overhead_ratio:1.08
rss_overhead_bytes:65323008
mem_fragmentation_ratio:1.12
mem_fragmentation_bytes:87792760
mem_not_counted_for_evict:0
mem_replication_backlog:0
mem_clients_slaves:0
mem_clients_normal:795104
mem_aof_buffer:0
mem_allocator:jemalloc-5.1.0
active_defrag_running:0
lazyfree_pending_objects:0

# Persistence
loading:0
rdb_changes_since_last_save:600161
rdb_bgsave_in_progress:0
rdb_last_save_time:1567699805
rdb_last_bgsave_status:ok
rdb_last_bgsave_time_sec:-1
rdb_current_bgsave_time_sec:-1
rdb_last_cow_size:0
aof_enabled:0
aof_rewrite_in_progress:0
aof_rewrite_scheduled:0
aof_last_rewrite_time_sec:-1
aof_current_rewrite_time_sec:-1
aof_last_bgrewrite_status:ok
aof_last_write_status:ok
aof_last_cow_size:0

# Stats
total_connections_received:35
total_commands_processed:615420
instantaneous_ops_per_sec:26490
total_net_input_bytes:631636742
total_net_output_bytes:4233130
instantaneous_input_kbps:3778.83
instantaneous_output_kbps:782.25
rejected_connections:0
sync_full:0
sync_partial_ok:0
sync_partial_err:0
expired_keys:0
expired_stale_perc:0.00
expired_time_cap_reached_count:0
evicted_keys:0
keyspace_hits:61077
keyspace_misses:0
pubsub_channels:0
pubsub_patterns:0
latest_fork_usec:0
migrate_cached_sockets:0
slave_expires_tracked_keys:0
active_defrag_hits:0
active_defrag_misses:0
active_defrag_key_hits:0
active_defrag_key_misses:0

# Replication
role:master
connected_slaves:0
master_replid:e9ea20433005cf1109e36ff4ccee8efa5922629e
master_replid2:0000000000000000000000000000000000000000
master_repl_offset:0
second_repl_offset:-1
repl_backlog_active:0
repl_backlog_size:1048576
repl_backlog_first_byte_offset:0
repl_backlog_histlen:0

# CPU
used_cpu_sys:2.019174
used_cpu_user:3.622899
used_cpu_sys_children:0.000000
used_cpu_user_children:0.000000

# Commandstats
cmdstat_set:calls=284807,usec=283381,usec_per_call=0.99
cmdstat_ai.modelrun:calls=15272,usec=50136,usec_per_call=3.28
cmdstat_ai.config:calls=1,usec=54295,usec_per_call=54295.00
cmdstat_flushall:calls=1,usec=1,usec_per_call=1.00
cmdstat_ai.tensorget:calls=15258,usec=44787,usec_per_call=2.94
cmdstat_ai.tensorset:calls=300080,usec=946506,usec_per_call=3.15
cmdstat_ai.modelset:calls=1,usec=34262,usec_per_call=34262.00

# Cluster
cluster_enabled:0

# Keyspace
db0:keys=600146,expires=0,avg_ttl=0

------ CLIENT LIST OUTPUT ------
id=36 addr=10.3.0.232:57768 fd=22 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=37 addr=10.3.0.232:57770 fd=23 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=22 addr=10.3.0.232:57746 fd=8 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=23 addr=10.3.0.232:57740 fd=9 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=24 addr=10.3.0.232:57741 fd=10 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=25 addr=10.3.0.232:57744 fd=11 name= age=1 idle=0 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=32768 obl=0 oll=0 omem=0 events=r cmd=ai.tensorget
id=26 addr=10.3.0.232:57752 fd=12 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=27 addr=10.3.0.232:57748 fd=13 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=28 addr=10.3.0.232:57750 fd=14 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=442 qbuf-free=32326 obl=5 oll=0 omem=0 events=r cmd=ai.modelrun
id=29 addr=10.3.0.232:57754 fd=15 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=30 addr=10.3.0.232:57760 fd=16 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=31 addr=10.3.0.232:57756 fd=17 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=32 addr=10.3.0.232:57758 fd=18 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=33 addr=10.3.0.232:57762 fd=19 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=34 addr=10.3.0.232:57764 fd=20 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun
id=35 addr=10.3.0.232:57766 fd=21 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=68 qbuf-free=32700 obl=0 oll=0 omem=0 events=r cmd=ai.modelrun

------ CURRENT CLIENT INFO ------
id=28 addr=10.3.0.232:57750 fd=14 name= age=1 idle=0 flags=b db=0 sub=0 psub=0 multi=-1 qbuf=442 qbuf-free=32326 obl=5 oll=0 omem=0 events=r cmd=ai.modelrun
argv[0]: 'AI.MODELRUN'
argv[1]: 'financialNet'
argv[2]: 'INPUTS'
argv[3]: 'transactionTensor:15272'
argv[4]: 'referenceTensor:15272'
argv[5]: 'OUTPUTS'
argv[6]: 'classificationTensor:15272'
16856:M 05 Sep 2019 16:11:11.480 # key 'financialNet' found in DB containing the following object:
16856:M 05 Sep 2019 16:11:11.480 # Object type: 5
16856:M 05 Sep 2019 16:11:11.480 # Object encoding: 0
16856:M 05 Sep 2019 16:11:11.480 # Object refcount: 1

------ REGISTERS ------
16856:M 05 Sep 2019 16:11:11.480 # 
RAX:0000000000000000 RBX:00007ffc442ca110
RCX:00007f4980000000 RDX:00007f499fa57d60
RDI:0000000000000000 RSI:00007f4a62c717b8
RBP:00007ffc442ca0d0 RSP:00007ffc442ca0b0
R8 :00007ffc442ca280 R9 :ffffffff00000000
R10:00007f4a62c71748 R11:000000000000003d
R12:00007ffc442ca0f0 R13:0000000000000001
R14:0000000000000000 R15:00007ffc442ca3a0
RIP:00007f4a5f863432 EFL:0000000000010202
CSGSFS:002b000000000033
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0bf) -> 0000000000000000
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0be) -> 0000000000000002
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0bd) -> 0000000000000000
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0bc) -> 0000000000000001
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0bb) -> 00007f4a5f866c16
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0ba) -> 00007ffc442ca2e0
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0b9) -> 00007f4a61810aa0
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0b8) -> 00007f4a61810b00
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0b7) -> 00007f4a5f866cdb
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0b6) -> 00007f4a61810500
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0b5) -> 00007f4a5f86735d
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0b4) -> 00007ffc442ca2e0
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0b3) -> 00007f499fa57d60
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0b2) -> 0000000000000001
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0b1) -> 00007f4a6180e280
16856:M 05 Sep 2019 16:11:11.480 # (00007ffc442ca0b0) -> 00007f4a61811040

------ FAST MEMORY TEST ------
16856:M 05 Sep 2019 16:11:11.480 # Bio thread for job type #0 terminated
16856:M 05 Sep 2019 16:11:11.480 # Bio thread for job type #1 terminated
16856:M 05 Sep 2019 16:11:11.480 # Bio thread for job type #2 terminated
*** Preparing to test memory region 562abbc72000 (2252800 bytes)
*** Preparing to test memory region 562abcf0b000 (135168 bytes)
*** Preparing to test memory region 562abcf2c000 (12984320 bytes)
*** Preparing to test memory region 7f4998000000 (655360 bytes)
*** Preparing to test memory region 7f499e800000 (562171904 bytes)
*** Preparing to test memory region 7f49c5000000 (50466816 bytes)
*** Preparing to test memory region 7f49cc600000 (60952576 bytes)
*** Preparing to test memory region 7f49d4400000 (63049728 bytes)
*** Preparing to test memory region 7f49dc400000 (63049728 bytes)
*** Preparing to test memory region 7f49e4180000 (4718592 bytes)
*** Preparing to test memory region 7f49e4700000 (59904000 bytes)
*** Preparing to test memory region 7f49ec000000 (135168 bytes)
*** Preparing to test memory region 7f49f0000000 (135168 bytes)
*** Preparing to test memory region 7f49f4000000 (135168 bytes)
*** Preparing to test memory region 7f49f8000000 (135168 bytes)
*** Preparing to test memory region 7f49fc1f9000 (6291456 bytes)
*** Preparing to test memory region 7f49fc7fa000 (8388608 bytes)
*** Preparing to test memory region 7f49fcffb000 (8388608 bytes)
*** Preparing to test memory region 7f49fd7fc000 (8388608 bytes)
*** Preparing to test memory region 7f49fdffd000 (8388608 bytes)
*** Preparing to test memory region 7f49fe7fe000 (8388608 bytes)
*** Preparing to test memory region 7f49fefff000 (8388608 bytes)
*** Preparing to test memory region 7f49ff800000 (8388608 bytes)
*** Preparing to test memory region 7f4a00000000 (135168 bytes)
*** Preparing to test memory region 7f4a04000000 (135168 bytes)
*** Preparing to test memory region 7f4a08000000 (135168 bytes)
*** Preparing to test memory region 7f4a0c000000 (135168 bytes)
*** Preparing to test memory region 7f4a10000000 (135168 bytes)
*** Preparing to test memory region 7f4a140f9000 (7340032 bytes)
*** Preparing to test memory region 7f4a147fa000 (8388608 bytes)
*** Preparing to test memory region 7f4a14ffb000 (8388608 bytes)
*** Preparing to test memory region 7f4a157fc000 (8388608 bytes)
*** Preparing to test memory region 7f4a15ffd000 (8388608 bytes)
*** Preparing to test memory region 7f4a167fe000 (8388608 bytes)
*** Preparing to test memory region 7f4a16fff000 (8388608 bytes)
*** Preparing to test memory region 7f4a17800000 (8388608 bytes)
*** Preparing to test memory region 7f4a18000000 (135168 bytes)
*** Preparing to test memory region 7f4a1c000000 (135168 bytes)
*** Preparing to test memory region 7f4a20000000 (135168 bytes)
*** Preparing to test memory region 7f4a24000000 (135168 bytes)
*** Preparing to test memory region 7f4a28000000 (135168 bytes)
*** Preparing to test memory region 7f4a2c000000 (135168 bytes)
*** Preparing to test memory region 7f4a30079000 (7864320 bytes)
*** Preparing to test memory region 7f4a307fa000 (8388608 bytes)
*** Preparing to test memory region 7f4a30ffb000 (8388608 bytes)
*** Preparing to test memory region 7f4a317fc000 (8388608 bytes)
*** Preparing to test memory region 7f4a31ffd000 (8388608 bytes)
*** Preparing to test memory region 7f4a327fe000 (8388608 bytes)
*** Preparing to test memory region 7f4a32fff000 (8388608 bytes)
*** Preparing to test memory region 7f4a33800000 (8388608 bytes)
*** Preparing to test memory region 7f4a34000000 (135168 bytes)
*** Preparing to test memory region 7f4a38000000 (135168 bytes)
*** Preparing to test memory region 7f4a3c000000 (135168 bytes)
*** Preparing to test memory region 7f4a40000000 (135168 bytes)
*** Preparing to test memory region 7f4a44000000 (135168 bytes)
*** Preparing to test memory region 7f4a48000000 (135168 bytes)
*** Preparing to test memory region 7f4a4c000000 (135168 bytes)
*** Preparing to test memory region 7f4a50152000 (5767168 bytes)
*** Preparing to test memory region 7f4a506d3000 (8388608 bytes)
*** Preparing to test memory region 7f4a50ed4000 (8388608 bytes)
*** Preparing to test memory region 7f4a516d5000 (8388608 bytes)
*** Preparing to test memory region 7f4a51ed6000 (8388608 bytes)
*** Preparing to test memory region 7f4a526d7000 (8388608 bytes)
*** Preparing to test memory region 7f4a52ed8000 (8388608 bytes)
*** Preparing to test memory region 7f4a536d9000 (8388608 bytes)
*** Preparing to test memory region 7f4a53eda000 (8388608 bytes)
*** Preparing to test memory region 7f4a54c77000 (16384 bytes)
*** Preparing to test memory region 7f4a5661c000 (69632 bytes)
*** Preparing to test memory region 7f4a5f5d0000 (491520 bytes)
*** Preparing to test memory region 7f4a5fa8b000 (8388608 bytes)
*** Preparing to test memory region 7f4a6028c000 (8388608 bytes)
*** Preparing to test memory region 7f4a60a8d000 (8388608 bytes)
*** Preparing to test memory region 7f4a61400000 (8388608 bytes)
*** Preparing to test memory region 7f4a6208c000 (16384 bytes)
*** Preparing to test memory region 7f4a622ab000 (16384 bytes)
*** Preparing to test memory region 7f4a62aa5000 (200704 bytes)
*** Preparing to test memory region 7f4a62c71000 (32768 bytes)
*** Preparing to test memory region 7f4a62c82000 (4096 bytes)
.O.O.O.O.O.O.O.O.O.O.O.O.O.O.O.O.
```

**Comment by filipecosta90** [comment]
tested against  brach prior-mq wich has commits up to (AI.MODELSET example TF would use a .pb file (#190) + perf improvement on RedisAI_Run_ThreadMain) and the crash does not happen @lantiga:
commit history of that branch:
```
commit 09d32d9c157641028334d2c749df2e8f36a5ffdd (HEAD -> prior-mq, origin/prior-mq)
Author: filipecosta90 <filipecosta.90@gmail.com>
Date:   Fri Sep 6 17:49:19 2019 +0100

    "[add] bump perf improvement on RedisAI_Run_ThreadMain"

commit 580c8169db242236faaa582302692a5736787e9a
Author: André Srinivasan <andre.srinivasan@gmail.com>
Date:   Tue Aug 27 00:17:28 2019 -0700

    AI.MODELSET example TF would use a .pb file (#190)
```

**Comment by lantiga** [comment]
Should have been hot-fixed by 04759c46
